readme.md

```diff
@@ -505,6 +505,9 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing.
 ## Other Awesome Lists
 
 - [awesome-git-addons](https://github.com/stevemao/awesome-git-addons) - A curated list of addons that extends git cli.
+- [awesome-macos-command-line](https://github.com/herrbischoff/awesome-macos-command-line) - A curated list of shell commands and tools specific to macOS.
```

Comment:
Please edit the link to end in `#readme`, which directs to the start of the readme instead of the start of the repo.
e.g.: `https://github.com/herrbischoff/awesome-macos-command-line#readme`